frontend/src/pages/IndexPage.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import Hero from '../components/Hero'
+import Sunscreens from '../components/Sunscreens'
+
+export default function IndexPage() {
+  const [sunscreens, setSunscreens] = useState([])
+    
+    useEffect(() => {
+      const getSunscreens = async () => {
+        const sunscreens = await fetchSunscreens()
+        setSunscreens(sunscreens)
+      }
+      getSunscreens()
+    }, [])
+
+    const fetchSunscreens = async () => {
+      const res = await fetch('http://localhost:2003/sunscreens')
+      const data = await res.json()
+      // console.log(data[0].name)
+      return data
+    }
+
+  return (
+    <>
+      <Hero />
+      <div>
+        <Sunscreens sunscreens={sunscreens} />
+      </div>
+    </>
+  )
+}
```

Comment:
Note these red circles with the horizontal lines through them, it's a small code formatting thing but it's best to have a blank line at the end of each file (the circles are denoting that there isn't a blank line). I can show you how to get automatic linting working to do this automatically!

Reply:
Good catch, thank you! Automatic linting would be a great help!